Description: Darken gradient used in GNOME Shell's top panel
 GNOME Shell's top panel is a bit too receptive to background colours when
 Zukitwo is used as the theme; i.e. when a light background is used, the
 panel tends to be too light in colour and makes it hard to read any text on
 the panel (which is white by default). Modify the gradient so that it is more
 dark, regardless of the darkness of the background.
Author: Vincent Cheng <Vincentc1208@gmail.com>
Last-Update: 2012-08-28

--- a/Zukitwo-Shell/gnome-shell/gnome-shell.css
+++ b/Zukitwo-Shell/gnome-shell/gnome-shell.css
@@ -482,8 +482,8 @@
 	border-right: 0px;
 	border-radius: 0px;
 	background-gradient-direction: vertical;
-	background-gradient-start: rgba(0,0,0,0.25);
-	background-gradient-end: rgba(0,0,0,0.45);
+	background-gradient-start: rgba(0,0,0,0.4);
+	background-gradient-end: rgba(0,0,0,0.8);
 	font-weight: bold;
 	height: 1.86em;
 }
